pdns/recursordist/docs/settings.rst

```diff
@@ -1356,11 +1356,15 @@ without consulting authoritative servers.
 .. versionadded:: 4.1.0
 
 -  Integer
--  Default: 2500
+-  Default: 150
 
 Maximum number of iterations allowed for an NSEC3 record.
 If an answer containing an NSEC3 record with more iterations is received, its DNSSEC validation status is treated as Insecure.
 
+.. versionchanged:: 4.5.2
```

Comment:
That changes will never appear in the online docs, right?

Reply:
indeed, that's why Otto noted above "Don't forget to forward port the docs changes!"

Reply:
ah!

Reply:
The plan is t include that commit into the prep for 4.5.2